### DISCUSSION TITLE
automatic letsencrypt autocert bot like golang?

### DISCUSSION THREAD

**Discussion by superdolt** [uNetworking/uWebSockets#1354] [Ideas]
nice work with uwebsocket
possible to do an automatic lets encrypt for this?

**Comment by joshxyzhimself** [comment]
This one is interesting.

Personally for simpler projects I just use caddy, quite simple and easy (less than 2 mins setup).

Example: https://gist.github.com/joshxyzhimself/d167c74d26610c53ebf642bb1d4556b5

For granular TLS configuration, HAProxy + Certbot

- https://caddyserver.com/
- https://www.haproxy.org/
- https://certbot.eff.org/
- https://ssl-config.mozilla.org/

**Comment by ghost** [comment]
Support for something like this would be very nice, I agree. But this falls under "out of scope" kind of features not part of this lib. Maybe someone could implement the support as some kind of extension?
